tests/test_files/project/PAYLIB.CBL

```diff
@@ -22,10 +22,10 @@
       * COPY REPL REPLACING ==TAG== BY ==ABC==
       * ==TAR== by ==XYZ==.
       * COPY REPL REPLACING ==TAG== BY DEF.
-       COPY PAYLIB REPLACING  A BY PAYROLL1
-                         B BY PAY-CODE
-                         C BY GROSS-PAY
-                         D BY HOURS.
+       COPY PAYLIB REPLACING LEADING  A BY PAYROLL1
```

Comment:
@ap891843 So, it is required to specify `LEADING` every time when the pattern is at the beginning of the line?

Reply:
Yes. 
use leading clause to match starts with for a word.
use trailing to match ends with for a word.
or use delimited COBOL word, like :ABC: for exact match